### PR TITLE
app-misc/trash-cli maintenance

### PR DESCRIPTION
**Pull request by zpuskas**
Drop old stable and fix QA issue in current version

### CODE REVIEW

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+HOMEPAGE="https://github.com/andreafrancia/trash-cli"
+SRC_URI="https://github.com/andreafrancia/${PN}/archive/${GIT_REF}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (juippis) [inline app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:23]: This is handled by `distutils-r1.eclass` for you.

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	nosetests -v || die
+}
```
Comment (juippis) [inline app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:31]: Looks to me like `distutils_enable_tests nose` should work okay here. You do need to add mock dependency separately though.

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1 vcs-snapshot
```
Comment (juippis) [inline app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:9]: I wonder if you really need *vcs-snapshot* eclass here, if you define `S`.

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild (hunk 1 above)
Comment (zpuskas) [reply app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:23]: TIL, will remove explicit dependency.

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild (hunk 3 above)
Comment (zpuskas) [reply app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:9]: Yeah, prolly we can get rid of this.

File: app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild (hunk 2 above)
Comment (zpuskas) [reply app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild:31]: Neat.

**Pull request closed without merging**